functional-tests/CloudbaseInit.Tests.ps1

```diff
@@ -287,6 +301,29 @@ function after.cloudbaseinit.plugins.windows.bootconfig.BootStatusPolicyPlugin {
 
 function prepare.nocloud {
     pushd "$here/../$($env:CLOUD)"
+
+        $openVpnUrl = "https://build.openvpn.net/downloads/releases/OpenVPN-2.5.10-I601-amd64.msi"
+        $wc = New-Object System.Net.WebClient
+        $msiFilePath = Join-Path $(pwd) "openvpn.msi"
+        $wc.DownloadFile($openVpnUrl, $msiFilePath)
+        cmd /c "msiexec.exe -i ${msiFilePath} /qn /norestart /l*v test.log"
+        Start-Sleep 10
+        if ($LASTEXITCODE) { throw "Failed to install openvpn" }
+        Get-NetAdapter | Out-Null
+        Start-Sleep 10

```

Comment:
What's the purpose of this sleep?

Reply:
Just for safety. I ll replace the whole thing with an exec retry

Reply:
the sleeps were removed, it seems there is no need for them.